chapter_crashcourse/ndarray.md

```diff
@@ -197,9 +199,9 @@ x += y
 id(x) == before
 ```
 
-## Mutual Transformation of NDArray and NumPy
+## `mxnet.numpy.ndarray` and `numpy.ndarray`
 
-Converting MXNet NDArrays to and from NumPy is easy. The converted arrays do *not* share memory. This minor inconvenience is actually quite important: when you perform operations on the CPU or one of the GPUs, you do not want MXNet having to wait whether NumPy might want to be doing something else with the same chunk of memory. The  `array` and `asnumpy` functions do the trick.
+Converting MXNet ndarrays to and from NumPy is easy. The converted arrays do **not** share memory. This minor inconvenience is actually quite important: when you perform operations on the CPU or one of the GPUs, you do not want MXNet having to wait whether NumPy might want to be doing something else with the same chunk of memory. The  `array` and `asnumpy` functions do the trick.
```

Comment:
Highlight that NumPy here is a scientific computing package of Python, not of MXNet.